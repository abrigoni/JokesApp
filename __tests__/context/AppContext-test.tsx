import React from 'react'
import {render} from '@testing-library/react-native'
import '@testing-library/jest-dom'
import AppContextProvider, { AppContext } from '../../src/context/AppContext';


describe('App Context', () => {
  test('Default state', () => {

  });
});
