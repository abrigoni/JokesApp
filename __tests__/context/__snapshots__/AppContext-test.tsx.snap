// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Context Renders correctly 1`] = `
<Text>
  Loading:
  0
</Text>
`;
